### DISCUSSION TITLE
Why was sortable-gallery removed from examples?

### DISCUSSION THREAD

**Discussion by yangchristina** [igordanchenko/react-photo-album#176] [Q&A] [answered]
Hi! I was wondering why `sortable-gallery` was removed from `examples` in v3. Is it no longer compatible?

**Comment by igordanchenko** [comment, marked as answer]
Hi there! I removed the `sortable-gallery` example because of the page scrolling issue on touch devices, which has been bothering me for a long time. But it looks like I can resolve that issue after all. I'll bring the `sortable-gallery` example back.

Can you share a demo of how you use a sortable gallery in a real-life project?

**Reply by yangchristina** [reply]
Great thanks! I haven't implemented a sortable gallery yet – was looking for a way to do so. I'd love to share that once I've added that feature though. I'm planning on using it as part of a form to create/reorder a photo collage.

**Reply by igordanchenko** [reply]
https://react-photo-album.com/examples/sortable-gallery

https://github.com/igordanchenko/react-photo-album/tree/main/examples/sortable-gallery
